src/main/scala/nl/knaw/dans/easy/solr/Defaults.scala

```diff
@@ -0,0 +1,37 @@
+package nl.knaw.dans.easy.solr
+
+import org.apache.commons.configuration.PropertiesConfiguration
+import scala.collection.JavaConverters._
+
+object Defaults {
+
+  /** maps long option names to the explicitly defined short keys */
+  val keyMap = new Conf().builder.opts
+    .withFilter(opt => opt.requiredShortNames.nonEmpty)
+    .map(opt => (opt.name, opt.requiredShortNames.head)).toMap
+
+  /**
+   * Gets defaults from properties for options that are not on the command line.
+   *
+   * @param args command line arguments
+   * @param props key-value pairs: if a key is prefixed with "default."
+   *              the rest of the key should equal one of the option names
+   * @return key-value pairs from props for keys not in args
+   */
+  def filter(args: Seq[String], props: PropertiesConfiguration): Seq[String] = {
```

Comment:
In my opinion the name `filter` doesn't clearly describe what the function does. Also because `filter` is commonly a higher-order function in Scala. Perhaps we could call it something like `getDefaultArgs`?